cffi/cffi_extension_build.py

```diff
@@ -0,0 +1,24 @@
+from cffi import FFI
+ffibuilder = FFI()
+
+ffibuilder.cdef("""\
+typedef struct Bigram {
+    int letter1AsciiCode;
+    int letter2AsciiCode;
+    double frequency;
+} Bigram;
+double test_single_layout(char* layout, int layout_length, Bigram* bigrams,
+                          int bigrams_count, double* score_list);
+void get_top_scores(char** layouts, long layout_count, double* scores, int nr_of_best,
+                    int letters_per_layer, char** top_layouts, double* top_scores);
+""")
```

Comment:
This is the same code as in the otther file, correct? Couldn't we read the file instead of having the code present in two places?

Reply:
I thought the same. I was wondering about the same type of duplication in the python-cffi [documentation](https://cffi.readthedocs.io/en/latest/overview.html#api-mode-calling-c-sources-instead-of-a-compiled-library). Maybe they do it like this in the example because it's only one line.
I'll change this.

Reply:
Done in 3247aff9871ed271bd5412607df638b6d869c05f.